src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {Animated, Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
 import MemoIc_Checklist from '../../../assets/icons/Ic_Checklist';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
 import {COLORS} from '../../../utils/theme';
 
-const ItemUser = ({photo, username, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
   const [isSelect, setIsSelect] = React.useState(false);
   const followIconFadeAnimation = React.useRef(new Animated.Value(0)).current;
+  const backgroundOpacity15PercentHexCode = '26';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new prop `isAnon` has been added to the `ItemUser` component. However, it's not clear from the code what the default value of `isAnon` should be if it's not provided. It would be good to have a default value for this prop to prevent any potential issues when it's not provided.
```diff
- const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
+ const ItemUser = ({photo, username, followed, onPress, userid, isAnon = false}) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->